It seems cann't be supported on android 8.0 devices.

### Describe the bug

the jellyfin client will be crashed when I try to open the setting option.
I found some error information in the log stack like "Only fullscreen opaque activities can request orientation".
my device's version is android 8.0.

### Logs

---
client: Jellyfin for Android TV
client_version: 0.15.5
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 
```log
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.jellyfin.androidtv/org.jellyfin.androidtv.ui.preference.PreferencesActivity}: java.lang.IllegalStateException: Only fullscreen opaque activities can request orientation
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2817)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2892)
	at android.app.ActivityThread.-wrap11(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1593)
	at android.os.Handler.dispatchMessage(Handler.java:105)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6541)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:775)
Caused by: java.lang.IllegalStateException: Only fullscreen opaque activities can request orientation
	at android.app.Activity.onCreate(Activity.java:986)
	at androidx.core.app.ComponentActivity.onCreate(ComponentActivity.java:88)
	at androidx.activity.ComponentActivity.onCreate(ComponentActivity.java:363)
	at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:217)
	at org.jellyfin.androidtv.ui.preference.PreferencesActivity.onCreate(PreferencesActivity.kt:11)
	at android.app.Activity.performCreate(Activity.java:6978)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1213)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2770)
	... 9 more

```  
***Logcat***: 
```log
03-28 22:26:21.567 D/ACRA    ( 6498): Adding reportSender : org.jellyfin.androidtv.telemetry.TelemetryService$AcraReportSender@f7ca6c5
03-28 22:26:21.568 I/ACRA    ( 6498): Sending report /data/user/0/org.jellyfin.androidtv/app_ACRA-approved/2023-03-28T22:26:17.829+08:00.stacktrace
03-28 22:26:21.590 D/ACRA    ( 6498): Sending report using org.jellyfin.androidtv.telemetry.TelemetryService$AcraReportSender
03-28 22:26:21.669 D/ACRA    ( 6498): Sent report using org.jellyfin.androidtv.telemetry.TelemetryService$AcraReportSender
03-28 22:26:21.669 D/ACRA    ( 6498): Report was sent by all senders
03-28 22:26:21.670 D/ACRA    ( 6498): Finished sending reports from SenderService
03-28 22:26:21.760 D/ACRA    ( 7458): ACRA processName='org.jellyfin.androidtv'
03-28 22:26:21.774 I/ACRA    ( 7458): ACRA is enabled for org.jellyfin.androidtv, initializing...
03-28 22:26:21.775 D/ACRA    ( 7458): SimplePluginLoader loading services from plugin classes : [Ljava.lang.Class;@e4c1953
03-28 22:26:21.776 D/ACRA    ( 7458): ServicePluginLoader loading services from ServiceLoader : java.util.ServiceLoader[org.acra.collector.Collector]
03-28 22:26:21.847 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.ConfigurationCollector
03-28 22:26:21.848 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.CustomDataCollector
03-28 22:26:21.848 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.DeviceFeaturesCollector
03-28 22:26:21.848 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.DeviceIdCollector
03-28 22:26:21.849 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.DisplayManagerCollector
03-28 22:26:21.850 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.DropBoxCollector
03-28 22:26:21.850 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.LogCatCollector
03-28 22:26:21.850 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.LogFileCollector
03-28 22:26:21.851 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.MediaCodecListCollector
03-28 22:26:21.851 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.MemoryInfoCollector
03-28 22:26:21.851 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.PackageManagerCollector
03-28 22:26:21.851 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.ReflectionCollector
03-28 22:26:21.851 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.SettingsCollector
03-28 22:26:21.852 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.SharedPreferencesCollector
03-28 22:26:21.852 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.SimpleValuesCollector
03-28 22:26:21.852 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.StacktraceCollector
03-28 22:26:21.852 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.ThreadCollector
03-28 22:26:21.862 D/ACRA    ( 7458): Loaded Collector of type org.acra.collector.TimeCollector
03-28 22:26:21.883 D/ACRA    ( 7458): SimplePluginLoader loading services from plugin classes : [Ljava.lang.Class;@e4c1953
03-28 22:26:21.883 D/ACRA    ( 7458): ServicePluginLoader loading services from ServiceLoader : java.util.ServiceLoader[org.acra.scheduler.SenderSchedulerFactory]
03-28 22:26:21.884 D/ACRA    ( 7458): SimplePluginLoader loading services from plugin classes : [Ljava.lang.Class;@e4c1953
03-28 22:26:21.884 D/ACRA    ( 7458): ServicePluginLoader loading services from ServiceLoader : java.util.ServiceLoader[org.acra.config.ReportingAdministrator]
03-28 22:26:21.905 I/LogInitializer( 7458): Debug tree planted
03-28 22:26:21.956 D/WM-WrkMgrInitializer( 7458): Initializing WorkManager with default configuration.
03-28 22:26:22.055 I/MigrationContext( 7458): Requested migration from 2 to 2. Found 1 migrations in total.
03-28 22:26:22.125 D/SessionRepositoryImpl( 7458): Restoring session
03-28 22:26:22.153 D/SessionRepositoryImpl( 7458): Updating current session. userId=d97a3bba-bc1b-494f-9014-08256c27c6ff apiBindingSuccess=true
03-28 22:26:22.163 I/org.jellyfin.sdk.api.client.KtorClient( 7458): GET http://192.168.8.8:8096/Users/Me
03-28 22:26:22.182 I/zygote  ( 7458): Do partial code cache collection, code=30KB, data=30KB
03-28 22:26:22.182 I/zygote  ( 7458): After code cache collection, code=28KB, data=28KB
03-28 22:26:22.182 I/zygote  ( 7458): Increasing code cache capacity to 128KB
03-28 22:26:22.196 I/ProcessLifecycleInitializer$create( 7458): Registering 0 ActivityLifecycleCallbacks
03-28 22:26:22.204 D/ACRA    ( 7458): ACRA processName='org.jellyfin.androidtv'
03-28 22:26:22.255 D/NetworkSecurityConfig( 7458): No Network Security Config specified, using platform default
03-28 22:26:22.256 D/AndroidRuntime( 7458): Shutting down VM
--------- beginning of crash
03-28 22:26:22.257 E/AndroidRuntime( 7458): FATAL EXCEPTION: main
03-28 22:26:22.257 E/AndroidRuntime( 7458): Process: org.jellyfin.androidtv, PID: 7458
03-28 22:26:22.257 E/AndroidRuntime( 7458): java.lang.RuntimeException: Unable to start activity ComponentInfo{org.jellyfin.androidtv/org.jellyfin.androidtv.ui.preference.PreferencesActivity}: java.lang.IllegalStateException: Only fullscreen opaque activities can request orientation
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2817)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2892)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at android.app.ActivityThread.-wrap11(Unknown Source:0)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1593)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at android.os.Handler.dispatchMessage(Handler.java:105)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at android.os.Looper.loop(Looper.java:164)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at android.app.ActivityThread.main(ActivityThread.java:6541)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at java.lang.reflect.Method.invoke(Native Method)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:775)
03-28 22:26:22.257 E/AndroidRuntime( 7458): Caused by: java.lang.IllegalStateException: Only fullscreen opaque activities can request orientation
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at android.app.Activity.onCreate(Activity.java:986)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at androidx.core.app.ComponentActivity.onCreate(ComponentActivity.java:88)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at androidx.activity.ComponentActivity.onCreate(ComponentActivity.java:363)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:217)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at org.jellyfin.androidtv.ui.preference.PreferencesActivity.onCreate(PreferencesActivity.kt:11)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at android.app.Activity.performCreate(Activity.java:6978)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1213)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2770)
03-28 22:26:22.257 E/AndroidRuntime( 7458): 	... 9 more
03-28 22:26:22.258 E/ACRA    ( 7458): ACRA caught a RuntimeException for org.jellyfin.androidtv
03-28 22:26:22.258 E/ACRA    ( 7458): java.lang.RuntimeException: Unable to start activity ComponentInfo{org.jellyfin.androidtv/org.jellyfin.androidtv.ui.preference.PreferencesActivity}: java.lang.IllegalStateException: Only fullscreen opaque activities can request orientation
03-28 22:26:22.258 E/ACRA    ( 7458): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2817)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2892)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at android.app.ActivityThread.-wrap11(Unknown Source:0)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1593)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at android.os.Handler.dispatchMessage(Handler.java:105)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at android.os.Looper.loop(Looper.java:164)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at android.app.ActivityThread.main(ActivityThread.java:6541)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at java.lang.reflect.Method.invoke(Native Method)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:775)
03-28 22:26:22.258 E/ACRA    ( 7458): Caused by: java.lang.IllegalStateException: Only fullscreen opaque activities can request orientation
03-28 22:26:22.258 E/ACRA    ( 7458): 	at android.app.Activity.onCreate(Activity.java:986)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at androidx.core.app.ComponentActivity.onCreate(ComponentActivity.java:88)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at androidx.activity.ComponentActivity.onCreate(ComponentActivity.java:363)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:217)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at org.jellyfin.androidtv.ui.preference.PreferencesActivity.onCreate(PreferencesActivity.kt:11)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at android.app.Activity.performCreate(Activity.java:6978)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1213)
03-28 22:26:22.258 E/ACRA    ( 7458): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2770)
03-28 22:26:22.258 E/ACRA    ( 7458): 	... 9 more
03-28 22:26:22.258 D/ACRA    ( 7458): Building report
03-28 22:26:22.259 D/ACRA    ( 7458): Starting collectors with priority FIRST
03-28 22:26:22.261 D/ACRA    ( 7458): Calling collector org.acra.collector.DropBoxCollector
03-28 22:26:22.262 D/ACRA    ( 7458): Calling collector org.acra.collector.LogCatCollector
03-28 22:26:22.262 D/ACRA    ( 7458): Collector org.acra.collector.DropBoxCollector completed
03-28 22:26:22.266 D/ACRA    ( 7458): Calling collector org.acra.collector.StacktraceCollector
03-28 22:26:22.267 D/ACRA    ( 7458): Collector org.acra.collector.StacktraceCollector completed
03-28 22:26:22.268 D/ACRA    ( 7458): Retrieving logcat output (buffer:default)...

```  

### App information

### Application version

0.15.5

### Where did you install the app from?

Sideloaded APK

### Device information

android tv

### Android version

android 8.0

### Jellyfin server version

10.8.9

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -138,6 +138,7 @@
 
         <activity
             android:name=".ui.preference.PreferencesActivity"
+            android:screenOrientation="behind"
             android:theme="@style/Theme.Jellyfin.Preferences" />
 
         <!-- Playback related activities -->

